src/logion/app.support.ts

```diff
@@ -48,7 +43,6 @@ export function predefinedSpec(spec: OpenAPIV3.Document): OpenAPIV3.Document {
 
     fillInSpecForProtectionController(spec);
     fillInSpecForTransaction(spec);
-    fillInSpecForAuthentication(spec);
```

Comment:
Why not keeping this line (and adapting the related import) ?
That would keep the resource `Authentication` documented under `/api-docs/`.
OK the code moved to another project, but the resource is still deployed the same way.

Reply:
This is actually done by `configureOpenApi`.